packages/system/cert-manager-issuers/templates/cluster-issuers.yaml

```
@@ -1,35 +1,56 @@
+            apiTokenSecretRef:  
+              name: cloudflare-api-token-secret  
+              key: api-token  
+      {{- else }}  
+        http01:  
+          ingress:  
+            class: nginx  
+      {{- end }}  
+
+---  
+
+apiVersion: cert-manager.io/v1  
```

yamllint

[error] trailing spaces

(trailing-spaces)